chalice/constants.py

```diff
@@ -190,6 +190,13 @@ def index():
 you to quickly create and deploy applications using
 Amazon API Gateway and AWS Lambda.
 
+This project is published as a preview project.
+Until the 1.0 GA release, we may introduce backwards
+incompatible changes based on customer feedback we receive.
+As a best practice, we do not recommend launching production
+APIs until 1.0 GA because of potential backward incompatible
```

Comment:
No I don't think so.   That warning has been in the README for almost a year now so I'd like to leave it as is.